counsel.el

```diff
@@ -3219,10 +3256,12 @@ substituted by the search regexp and file, respectively.  Neither
   (or
    (ivy-more-chars)
    (let* ((regex (counsel--grep-regex string))
-          (cmd (format counsel-grep-command (shell-quote-argument regex))))
+          (cmd (counsel--format
+                counsel-grep-command
+                (shell-quote-argument regex))))
```

Comment:
Is this shell quoting necessary even when `counsel-grep-command` is a list?

Reply:
No need, thanks